service/src/main/java/com/myhome/configuration/properties/mail/MailProperties.java

```diff
@@ -0,0 +1,48 @@
+package com.myhome.configuration.properties.mail;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+
+@Data
+@ConfigurationProperties(prefix = "spring.mail")
+public class MailProperties {
+
+  private String host;
+  private String username;
+  private String password;
+  private int port;
+  private String protocol;
+  private boolean debug;
+  private boolean devMode;
+  private boolean testConnection;
+
+  private MailTemplatesLocalization localization = new MailTemplatesLocalization();
```

Comment:
As mentioned in https://github.com/jmprathab/MyHome/pull/216#discussion_r579852405 we should have two other types of classes to store those properties.
For now, `EmailTemplateProperties` and `EmailTemplateLocalizationProperties` seems to be a good names for them.


For every property class please add @Component annotation to make it visible for Spring context during runtime (issue that you mentioned in https://github.com/jmprathab/MyHome/pull/216#discussion_r580606771)

Reply:
@mslowiak `@Component` helped with integration test now I can delete test configuration